app/scripts/filters/util.js

```diff
@@ -348,6 +348,30 @@ angular.module('openshiftConsole')
           .replace(/^./, function(str){ return str.toUpperCase(); });
     };
   })
+  // event messages should simple, but human readable:
+  // Pulled Pod <name>
+  // Unhealthy Pod <name>
+  // Deployment created for DeploymentConfig <name>
+  // etc.
+  // This filter handles the "Reason <optional preposition>"
+  // but not the <name> as we <a> link that in templates.
+  .filter('messageForEvent', function($filter) {
```

Comment:
Example of the updated language:

![screen shot 2017-08-14 at 11 12 19 am](https://user-images.githubusercontent.com/280512/29280358-ec97f394-80e8-11e7-9644-a44904d2fe8f.png)

The test file uses every Kind/Reason in our whitelist.  The templates are simple, but we can update/tweak if we feel it necessary.